lib/flutter_camera_ml_vision.dart

```diff
@@ -152,18 +145,20 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
   }
 
   CameraValue get cameraValue => _cameraController?.value;
+
   ImageRotation get imageRotation => _rotation;
 
   Future<void> Function() get prepareForVideoRecording =>
       _cameraController.prepareForVideoRecording;
 
-  Future<void> startVideoRecording(String path) async {
+  Future<void> startVideoRecording() async {
     await _cameraController.stopImageStream();
-    return _cameraController.startVideoRecording(path);
+    return _cameraController.startVideoRecording();
```

Comment:
The function return type is Future<void> look like the is a type mismatch

Reply:
I don't understand the comment here, _cameraController.startVideoRecording return a future so look fine to me

Reply:
does _cameraController.startVideoRecording return Future<void> ?

Reply:
Yep it does :)